spring-cloud-stream-schema/pom.xml

```diff
@@ -47,4 +49,27 @@
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.avro</groupId>
+				<artifactId>avro-maven-plugin</artifactId>
+				<version>${avro.version}</version>
+				<executions>
+					<execution>
+						<phase>generate-sources</phase>
```

Comment:
Use `generate-test-sources` as we will want to confine source generation strictly to tests.